include/seqan3/std/ranges

```diff
@@ -107,6 +130,23 @@ inline constexpr bool enable_borrowed_range<T> = true;
 
 #include <seqan3/std/concepts>
 #include <seqan3/std/iterator>
+#include <string_view>
+
+// until range <= 0.11.0 release, std::basic_string_view was not recognised as view or borrowed_range
+#if RANGE_V3_VERSION <= 1100
+#if defined(__cpp_lib_string_view) && __cpp_lib_string_view == 201603L
+namespace ranges
+{
+template<typename Char, typename Traits>
+    requires true
+inline constexpr bool enable_view<std::basic_string_view<Char, Traits>> = true;
+
+template<class CharT, class Traits>
+    requires true
+inline constexpr bool enable_borrowed_range<std::basic_string_view<CharT, Traits>> = true;
+} // namespace ranges
+#endif // defined(__cpp_lib_string_view) && __cpp_lib_string_view >= 201603L
+#endif // RANGE_V3_VERSION <= 1100
```

Comment:
As far as I can see, ranges-v3 0.11.0 does this already:

https://github.com/ericniebler/range-v3/blob/0.11.0/include/range/v3/range/concepts.hpp#L186-L190
and 
https://github.com/ericniebler/range-v3/blob/0.11.0/include/range/v3/range/access.hpp#L43-L47

Did I miss something?

Or did you mean to say `RANGE_V3_VERSION < 1100`? But then: we require 0.11.0

Reply:
Interesting: the current master does have an error in it:

https://github.com/ericniebler/range-v3/blob/b0696c75dd8b420432c76e5d90fa133ccb423437/include/range/v3/range/access.hpp#L45-L49

That means the version should be `RANGE_V3_VERSION >= 1100`.

I proposed a patch here https://github.com/ericniebler/range-v3/pull/1625 for current master.

Reply:
Now this makes sense :)
Depends on when this is fixed upstream. `>=` would be the most secure bet for now.

Reply:
yep